libc/config/baremetal/arm/entrypoints.txt

```diff
@@ -76,6 +69,15 @@ set(TARGET_LIBC_ENTRYPOINTS
     libc.src.string.strtok_r
     libc.src.string.strxfrm
 
+    # strings.h entrypoints
+    libc.src.strings.bcmp
+    libc.src.strings.bcopy
+    libc.src.strings.bzero
+    libc.src.strings.index
+    libc.src.strings.rindex
+    libc.src.strings.strcasecmp
+    libc.src.strings.strncasecmp
```

Comment:
@petrhosek so if your goal is to avoid POSIX in baremetal, then we can make llvm-libc slightly smaller for baremetal by excluding bcopy, index, rindex, strcasecmp, and strncasecmp which are POSIX and not ISO C.  LLVM may emit libcalls to bcmp (IMO) and bzero (I suspect) (unsure about bcopy, @gchatelet may know; otherwise I know where to look in llvm but am lazy).

Reply:
Clang needs to know that these functions [are present](https://github.com/llvm/llvm-project/blob/0f7b3a9407d20e6a4d33ea623e05cf2a3f65eabd/llvm/lib/Analysis/TargetLibraryInfo.cpp#L111-L119). And they are when using glibc or musl libc in the target triple.